src/optimizer/plugins/AddPathRulesOptimizer.ts

```diff
@@ -0,0 +1,61 @@
+import { Package } from '../../processor';
+import { OptimizerPlugin } from '../OptimizerPlugin';
+import SimplifyArrayIndexingOptimizer from './SimplifyArrayIndexingOptimizer';
+import { ProcessingOptions } from '../../utils';
+import { ExportablePathRule, ExportableSdRule } from '../../exportable';
+import { cloneDeep } from 'lodash';
+import SimplifyRulePathContextsOptimizer from './SimplifyRulePathContextsOptimizer';
+
+export default {
+  name: 'add_path_rules',
+  description: 'Add path rules to support indenting.',
+  runBefore: [SimplifyArrayIndexingOptimizer.name, SimplifyRulePathContextsOptimizer.name],
```

Comment:
I assume you're talking about `ConstructInlineInstanceOptimizer`? If so, yeah, I think that makes sense because we'd also want those new inline instances to have proper indentation.

Reply:
Oh yep, I was. I wrote this comment right after looking at that file and didn't write down the context my head was in.

Reply:
Added an `after` clause in 112d742.